app/controllers/admin/project_logs_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::ProjectLogsController < Admin::AdminController
+  def destroy
+    @log = ProjectLog.find(params[:id])
+    @step = ProjectStep.find(@log.project_step_id)
+    @loan = Loan.find(@step.project_id)
+
+    if @log.destroy
+      redirect_to admin_loan_path(@loan), notice: I18n.t(:notice_deleted)
```

Comment:
I think if you do `redirect_to admin_loan_path(@loan, anchor: 'timeline')...` it will redirect you to the timeline tab, but I'm not sure what the easiest way is to make sure that the correct step's log view is re-opened.